src/styles/blocks/card.scss

```diff
@@ -0,0 +1,92 @@
+.card {
+  width: 200px;
+  display: $flex-display;
+  flex-direction: column;
+  box-sizing: border-box;
```

Comment:
move to `*` selector

Reply:
it does not work, if i move 2-5 rows, so i moved 5-th row